draft-ietf-rats-architecture.md

```diff
@@ -412,8 +420,119 @@ implicitly trusted is often referred to as a Root of Trust.
 
 # Conceptual Messages {#messages}
 
-    <this section can include content from Serialization Formats and Conceptual Messages sections from
-    draft-thaler-rats-architecture, and Role Messages content from draft-birkholz-rats-architecture>
+## Evidence
+
+Today, Evidence tends to be highly device-specific, since the information in the Evidence
+often includes vendor-specific information that is necessary to fully describe the manufacturer
+and model of the device including its security properties, the health
+of the device, and the level of confidence in the correctness of the information.
+Evidence is typically signed by the device (whether by hardware, firmware, or software on the
+device), and evaluating it in isolation would require Appraisal Policy to be based on
+device-specific details (e.g., a device public key).
+
+## Endorsements
+
+An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
+device's signing capability.  For example, if the signing capability is in hardware, then
+an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
+private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+are used together, evaluating them can be done against Appraisal Policy that may not be specific to the
+device instance, but merely specific to the manufacturer providing the Endorsement. For example,
+an Appraisal Policy might simply check that devices from a given manufacturer have information
+matching a set of known-good reference values, or an Appraisal Policy might have a set of more complex
+logic on how to evaluate the validity of information.
+
+However, while an Appraisal Policy that treats all devices from a given manufacturer the same
+may be appropriate for some use cases, it would be inappropriate to use such an Appraisal Policy
+as the sole means of authorization for use cases that wish to constrain *which* compliant devices
+are considered authorized for some purpose.  For example, an enterprise using attestation for
+Network Endpoint Assessment may not wish to let every healthy laptop from the same
+manufacturer onto the network, but instead only want to let devices that it legally owns
+onto the network.  Thus, an Endorsement may be helpful information in authenticating
+information about a device, but is not necessarily sufficient to authorize access to
+resources which may need device-specific information such as a public key for the device or
+component or user on the device.  
```

Comment:
It seems that the point of this paragraph is that appraisal is a key part of authorization, but often not all.